packages/Video/Video.jsx

```diff
@@ -347,6 +360,12 @@ class Video extends React.Component {
 
   // ******** Begin Helper Functions *********
 
+  showWarnings = () => {
```

Comment:
This is not needed. The `videoBorder` prop can be used as long as the video has a white background.